vimeo-networking/src/main/java/com/vimeo/networking/model/TextTrack.java

```diff
@@ -0,0 +1,116 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Created by Mohit Sarveiya on 6/29/17.
+ */
+@UseStag
+@SuppressWarnings({"unused", "WeakerAccess"})
+public class TextTrack implements Serializable {
+
+    private static final long serialVersionUID = 3019422318347933808L;
+
+    @SerializedName("uri")
+    protected String mUri;
```

Comment:
Stag supports private visibility on fields as long as a setter/getter is present, and since I see that you have getters and setters for all fields, this should work fine as private. Also, while you're at it, could you update stag to version 2.3.1 (it's currently at 2.2.0).